tests/mocks/test_account_creation_policy.py

```diff
@@ -35,6 +35,33 @@ def test_modifying_account_creation_policy(self):
                     AccountCreationPolicy.EMAIL_STATUS_ENABLED
             })
 
+    def test_modifying_email_domain_whitelist_and_blacklist(self):
```

Comment:
@rdegges for some reason, this warning about Method could be a function still lingers after I've refactored the mock. It should be fine now (when I click on codacy's warning, it says "0 issues" and "2 duplications" although I don't see it and codacy doesn't highlight any code as problematic).